CHANGELOG.md

```diff
@@ -4,12 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```

Comment:
Can you add a new line of [Unreleased] at the top? This way we can start marking what *is not* in this release?

Reply:
Sounds good! was wondering about that 😄